src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw

```diff
@@ -173,7 +173,11 @@
     <value>Found a positional argument when none was expected</value>
   </data>
   <data name="FeatureDisabledMessage" xml:space="preserve">
-    <value>This command is a work in progress, and may be changed dramatically or removed altogether in the future. To enable it, edit your settings ('winget settings') to include the experimental feature</value>
+    <value>This feature is a work in progress, and may be changed dramatically or removed altogether in the future. To enable it, edit your settings ('winget settings') to include the experimental feature</value>
+    <comment>{Locked="winget settings"}</comment>
+  </data>
+  <data name="FeatureDisabledMessageWithFeatureName" xml:space="preserve">

```

Comment:
>FeatureDisabledMessageWithFeatureName [](start = 14, length = 37)

The other value is already used this way, just as the exception message that gets output as:

<message>: <parameter>

Don't make a second one.